test/e2e/dra/kind.yaml

```diff
@@ -42,3 +42,4 @@ nodes:
         v: "5"
 featureGates:
   DynamicResourceAllocation: true
+  DRAResourceClaimDeviceStatus: true
```

Comment:
is there a tab or some indent problem

```
++ for feature in ${features}
++ echo '  DRAResourceClaimDeviceStatus: true'
ERROR: failed to create cluster: unable to decode config: yaml: unmarshal errors:
  line 46: mapping key "DRAResourceClaimDeviceStatus" already defined at line 45
+ kind export logs /logs/artifacts/kind
```

Reply:
I think the feature-gate is duplicated, the CI seems to check the feature-gates by itself:
```
 features=($(grep '"DRA' pkg/features/kube_features.go | sed 's/.*"\(.*\)"/\1/'))
++ grep '"DRA' pkg/features/kube_features.go
++ sed 's/.*"\(.*\)"/\1/'
+ echo 'Enabling DRA feature(s): DRAResourceClaimDeviceStatus.'
Enabling DRA feature(s): DRAResourceClaimDeviceStatus.
```

Reply:
`pull-kubernetes-kind-dra-all` does detect the feature gates but not `pull-kubernetes-kind-dra`

Reply:
I fixed it by adding the feature-gate for that new test. `pull-kubernetes-kind-dra` is only for core DRA and `pull-kubernetes-kind-dra-all` should set the feature-gate automatically and run that new test.

Reply:
Right, I checked with him on private message on Slack earlier today (I should have mention it in my previous message sorry).

Reply:
From chat:
- don't change kind.yaml
- `pull-kubernetes-kind-dra-all` automatically adds all defined DRA feature gates
- label tests depending on new features with `feature.DRA<my new feature gate>` to ensure that it only runs in `pull-kubernetes-kind-dra-all`